packages/quantic/force-app/main/default/lwc/quanticNoResults/quanticNoResults.html

```diff
@@ -0,0 +1,29 @@
+<template>
+    <template if:false={hideNoResultsPanel}>
+        <div class="slds-grid slds-grid_vertical">
+            <div class=" slds-col slds-size_12-of-12 slds-var-m-vertical_x-large slds-illustration slds-illustration_small">
+                <c-quantic-svg name="noResults"></c-quantic-svg>
+            </div>
+            <div class="slds-col slds-size_12-of-12">
+                <h3 class="slds-text-heading_small slds-text-align_center slds-var-m-around_small">
+                    <lightning-formatted-rich-text value={noResultsTitleLabel}></lightning-formatted-rich-text>
+                </h3>
+                <p class="slds-text-body_small slds-text-align_center">
+                    <template if:true={hasBreadcrumbs}>
+                        <lightning-formatted-rich-text value={labels.noResultsWithFilters}></lightning-formatted-rich-text>
+                    </template>
+                    <template if:false={hasBreadcrumbs}>
+                        <lightning-formatted-rich-text value={labels.noResultsWithoutFilters}></lightning-formatted-rich-text>
+                    </template>
+                </p>
+            </div>
+            <template if:false={disableCancelLastAction}>
```

Comment:
to avoid using two template ifs I think we could combine these boolean variables into a condition that makes sense here. Something like `showUndoButton` perhaps?